src/bin/cql-stress-cassandra-stress/settings/option/rate.rs

```diff
@@ -87,13 +87,21 @@ impl RateOption {
                 max_threads,
                 auto,
             },
-            _ => ThreadsInfo::Fixed {
-                // SAFETY: The parameters are grouped in a way that this won't ever panic
-                // when entering this branch.
-                threads: threads.unwrap(),
-                throttle,
-                co_fixed,
-            },
+            _ => {
+                let (final_throttle, co_fixed) = match (fixed_rate, throttle) {
```

Comment:
it is not `fixed_rate`, it is `fixed coordinated omission latencies when rate is specified`.
`throttle` here is `coordinated omission affected latencies when rate is specified.`

See https://www.scylladb.com/2021/04/22/on-coordinated-omission/

Reply:
I still think it's fixed rate: user defines if cql-stress should just throttle to specified value or if is trying to keep fixed rate and show increased latencies if system can't keep up (taking into account delay in sending request from schedule).

Reply:
Look at the request scheduler in the code if you don't believe me.
Request flow is completely the same for both - `throttle` and `fixed` (`co_fixed`) parameters.
Latencies calculation is `the only` difference and reason for having two parameters instead of one.

Reply:
still we are discussing user facing part, while internals didn't change, I don't understand the problem

Reply:
`fixed_rate` is missleading, because we have the same static number of op/s in both cases.

Reply:
ok, I don't mind changing this local var name, changed to fixed